cursorless-talon-dev/src/spoken_form_test.py

```diff
@@ -10,6 +10,13 @@
     "Used to run tests on the Cursorless spoken forms/grammar",
 )
 
+mod.setting(
```

Comment:
```suggestion
# This setting is necessary because we can't detect if you're using "System Default", as `actions.sound.active_microphone()` will return what microphone that is pointing to.
mod.setting(
```